pkgs/games/anki/default.nix

```diff
@@ -1,199 +1,383 @@
-{ stdenv
-, buildPythonApplication
-, lib
-, python
-, fetchurl
+{ lib
+, stdenv
+, bash
+, buildEnv
 , fetchFromGitHub
-, fetchpatch
+, fetchYarnDeps
+, fixup_yarn_lock
 , lame
 , mpv-unwrapped
-, libpulseaudio
-, pyqtwebengine
-, decorator
-, beautifulsoup4
-, sqlalchemy
-, pyaudio
-, requests
-, markdown
-, matplotlib
-, mock
-, pytest
-, glibcLocales
-, nose
-, jsonschema
-, setuptools
-, send2trash
+, ninja
+, nodePackages
+, nodejs
+, nodejs-slim
+, pkg-config
+, protobuf
+, python39
+, qt6
+, rsync
+, rustPlatform
+, symlinkJoin
+, writeShellScriptBin
+, yarn
 , CoreAudio
-  # This little flag adds a huge number of dependencies, but we assume that
-  # everyone wants Anki to draw plots with statistics by default.
-, plotsSupport ? true
```

Comment:
this used to pull in matplotlib, but grepping around in the repo, it looks like they don't use that anymore, so we can drop this.